### ISSUE TITLE
ReplayKit throws exception when starting

### ISSUE THREAD

**Issue by syberkitten** [rainbowcreatures/FlashyWrappers#27]
Been trying the experimental replay kit,
when i start it (after loading and getting the ready event):
I get this error:
`
[Fault] exception, information=Error: [FlashyWrappers error] Error Domain=com.apple.ReplayKit.RPRecordingErrorDomain Code=-5803 "(null)"
`

**Comment by rainbowcreatures** [comment]
Hey, it seems some devices are not supported by ReplayKit, regardless of iOS version:

https://spryfox.zendesk.com/hc/en-us/articles/219399537-What-are-the-requirements-for-using-ReplayKit-iOS-devices-only-

**Comment by rainbowcreatures** [comment]
Yeah, its strange. I've got iPad mini 2 so I never realized there is such a list, until I googled for it just now. I thought it would be limited by iOS version as usual... I don't relly understand why they did it, probably they were worried recording on older devices will limit performance and they didn't want to deal with support caused by that.

"Normal" recording doesn't work with sound by default (it doesn't record audio magically). You'd need to use methods to add soundtracks or use FWSoundMixer (also an open source project here) to do your own sound mixing by playing virtual sounds and sending them to FW. 

Pause/resume is not supported as well.
